internal/cli/atlas/clusters/update.go

```diff
@@ -85,6 +86,19 @@ func (opts *UpdateOpts) cluster() (*atlasv2.AdvancedClusterDescription, error) {
 		if opts.name == "" {
 			opts.name = cluster.GetName()
 		}
+
+		// The cluster name cannot be updated by the Update operation
+		if cluster.GetName() != "" && opts.name != cluster.GetName() {
+			cluster.Name = nil
+			_, _ = log.Warning("the `name` field cannot be updated by the update operation. Ignoring `name` field...\n")
+		}
+
+		// The connection string cannot be updated by the Update operation
+		if cluster.ConnectionStrings != nil {
+			cluster.ConnectionStrings = nil
+			_, _ = log.Warning("\nthe `connectionStrings` field cannot be updated by the update operation`. Ignoring `connectionStrings` field...\n")
+		}
```

Comment:
can you move these to `removeReadOnlyAttributes` and drop the warnings?

Reply:
Updated. I cannot move `getName` as I need the `opts` to check that the provided name in the file is different than the one provided as an argument. This is a specific scenario of the update command.